.github/workflows/maven.yml

```diff
@@ -0,0 +1,21 @@
+name: Java CI with Maven
+
+on:
+  push:
+    branches:
+      - main
```

Comment:
Danke fürs aufsetzen!

In anderen Projekten habe ich die nachfolgende Einstellung gefunden (bzw. so von Michal übernommen), damit mMn vor dem mergen nach main (damals auch noch master) getested wird. Dann müsste der PR den Test triggern. Oder tun es auch deine Einstellungen schon per default?

> on:
>   push:
>     branches-ignore:
>       - 'master' # no need to run after merging to master
>       - 'main' # no need to run after merging to main; main is the "new" master
>     pull_request:

https://github.com/matsim-vsp/logistics/blob/442cf153ef99bed645f8e8fa2692cb8434126251/.github/workflows/GithubActionsCI_maven.yml#L6C1-L12C1

Reply:
Danke, guter Hinweis! Hab's hinzugefügt.